result.json

```diff
@@ -0,0 +1,207 @@
+{
```

Comment:
is this file needed?

Reply:
No, sorry. I added it just so that folks could see the result of running the script, it would be removed before merging in any case :+1: